.github/workflows/promote.yml

```diff
@@ -118,7 +118,8 @@ jobs:
 
   promote-infra-dev:
     needs: [build-prisma-client-lambda-layer, unit-tests]
-    uses: CMSgov/managed-care-review/.github/workflows/deploy-infra-to-env.yml@main
+    # TODO update inputs/secrets and pin back to main after verifying OIDC in dev
+    uses: CMSgov/managed-care-review/.github/workflows/deploy-infra-to-env.yml@10f745b1ba147dbef9b4161bb41d9c4aa1b4f2b9
```

Comment:
Do these need to have the environment: tags set to work?

Reply:
Sort of, @macrael. Once the pin is moved back to `main`, then environment is a required input for `deploy-infra-to-env`.  The way you do environments and environment secrets with reusable workflows is weird. The [tldr; in this article](https://colinsalmcorner.com/consuming-environment-secrets-in-reusable-workflows/#tldr) is good at clarifying this.